File: backend/components/metiers/administration_publique/guides/helpers/services/helpers_services.test.py
# helpers_services.test.py – Tests unitaires et edge cases Python
from .helpers_services import check_service, audit_service

def test_check_service():
    assert check_service({'status': 'ok', 'uptime': 99}) is True
    assert check_service({'status': 'fail', 'uptime': 10}) is False

def test_audit_service():
    result = audit_service({'status': 'ok', 'uptime': 99})
    assert isinstance(result, dict)
    assert 'score' in result
    assert 'compliant' in result

File: backend/components/metiers/administration_publique/guides/helpers/services/helpers_services.test.js
// helpers_services.test.js – Tests unitaires et edge cases JS
const { checkService, auditService } = require('./helpers_services');

test('checkService returns true for ok', () => {
  expect(checkService({ status: 'ok', uptime: 99 })).toBe(true);
  expect(checkService({ status: 'fail', uptime: 10 })).toBe(false);
});

test('auditService returns audit object', () => {
  const result = auditService({ status: 'ok', uptime: 99 });
  expect(typeof result).toBe('object');
  expect(result).toHaveProperty('score');
  expect(result).toHaveProperty('compliant');
});
